views/active-directors.html

```diff
@@ -2,54 +2,59 @@
 
 {% block pageTitle %}
 {% if errorList.length > 0 %}
-  Error:
-{% endif %}  Review the officers
+Error:
+{% endif %} Review the officers
 {% endblock %}
 
 {% block backLink %}
-  {% include "includes/back-link.html" %}
+{% include "includes/back-link.html" %}
 {% endblock %}
 
 {% block content %}
-  <div class="govuk-grid-row">
-    <div class="govuk-grid-column-three-quarters">
-      <form action="" method="post">
-        {% if officerErrorMsg %}
-          {{ govukErrorSummary({
-            titleText: "There is a problem",
-            errorList: [
-              {
-                text: officerErrorMsg,
-                href: "#active-directors"
-            }
-          ]
-          }) }}
-          {% set activeOfficersError = {
-            text: officerErrorMsg
-          }
-          %}
-        {% else %}
-          {% set activeOfficersError = false %}
-        {% endif %}
+<div class="govuk-grid-row">
+  <div class="govuk-grid-column-three-quarters">
+    <form action="" method="post">
+      {% if officerErrorMsg %}
+      {{ govukErrorSummary({
+      titleText: "There is a problem",
+      errorList: [
+      {
+      text: officerErrorMsg,
+      href: "#active-directors"
+      }
+      ]
+      }) }}
+      {% set activeOfficersError = {
+      text: officerErrorMsg
+      }
+      %}
+      {% else %}
+      {% set activeOfficersError = false %}
+      {% endif %}
```

Comment:
you did what i did and copied and pasted it in, which gets rid of the tabbing. need to add it back in.

Reply:
Actually I let VS format it and this is the mess it made